runtime/lua/vim/lsp/util.lua

```diff
@@ -757,7 +757,7 @@ do
     local lines = {"Diagnostics:"}
     local highlights = {{0, "Bold"}}
     local line_diagnostics = M.get_line_diagnostics()
-    if not line_diagnostics then return end
+    if not next(line_diagnostics) then return end
```

Comment:
The meaning is the same, but we are using it within Nvim.
```suggestion
    if not vim.tbl_isempty(line_diagnostics) then return end
```

Reply:
Yeah, this is better I think.

Reply:
Hmm, that does not seem to be working for me. It still seems to be showing an empty window when testing this method

Reply:
oops sorry. should delete "not"
`if vim.tbl_isempty(line_diagnostics) then return end`

Reply:
Can you modify it to print the inspected version of `line_diagnostics`? I'm wondering if there is something there somehow?

Reply:
Oh, haha, yes, h-michael's suggestion is better